include/tvm/script/printer/doc.h

```diff
@@ -219,6 +292,7 @@ class IdDoc : public ExprDoc {
    * \param name The name of identifier.
    */
   explicit IdDoc(String name);
+  explicit IdDoc(std::nullptr_t) : ExprDoc(nullptr) {}
```

Comment:
Yeah this is a good idea to explicitly define constructor with `std::nullptr_t` so that the compiler could distinguish between `nullptr -> const char* -> String` and `nullptr -> ObjectPtr`